lib/remove_duplicate_emails.rb

```diff
@@ -0,0 +1,39 @@
+class RemoveDuplicateEmails
+  def run(do_it: false, older_than: '1/10/2020' )
+    # find all duplicate contact infos, older than X
+    all_dup_contact_infos = ContactInfo.
+      where("created_at < ?", Date.parse(older_than)).
```

Comment:
Might consider using `Date.strptime(older_than, "%d/%m/%Y")` to make it a bit more specific.

Reply:
I would say make it YYYY-mm-dd so there's absolutely no chance for confusion.

Reply:
That works - I ask because I was initially confused 😄

Reply:
@mwharrison only in USA do we get confused on stuff like dates and measuring distance and weight :)))